### DISCUSSION TITLE
Content Not showing on preview

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10635] [Q&A]
Create content on admin side but when i preview that It not show any thing I working with decouple cms:
Anyone share with me the syntax so it call properly the flow based page in .cshtml:

Call like that synatx below:

@page "/article/{alias}"
@inherits OrchardCore.DisplayManagement.RazorPages.Page

@{
    var article = await Orchard.GetContentItemByAliasAsync(Model.Alias);
    ViewData["Title"] = article.DisplayText;
}

<h1>@article.DisplayText</h1>
    var blogPost = await Orchard GetContentItemByIdAsync(Id);
    <p> @Html.Raw(await Orchard.MarkdownToHtmlAsync((string)article.Content.MarkdownBodyPart.Markdown));

    @functions {
        [FromRoute]
        public string Alias { get; set; }

    }

but the flow section not properly show:

![issue-template](https://user-images.githubusercontent.com/5202291/140424889-aa1e7d47-be1d-42fc-81b4-e40e5ff2985b.png)

**Comment by Piedone** [comment]
You'll need to add and configure Preview Part for the content type for Content Preview to work in a decoupled app.

**Reply by saleemhaq** [reply]
Please advise based on screen shot below I unable to achieve?

![image](https://user-images.githubusercontent.com/5202291/140985684-d118bdc2-8fff-4fd4-a7a6-9ba332c7b6d1.png)
